.github/workflows/test-library.yml

```
@@ -195,6 +194,11 @@ jobs:
             && github.event.inputs.release-version
             || steps.scm-version.outputs.dist-version
         }}-py3-none-any.whl')
+    - name: Set PROXYPY_CONTAINER_VERSION env
+      shell: bash
+      run: >-
+        VER=$(echo '${{ needs.pre-setup.outputs.dist-version }}' | tr + .);
+        echo "PROXYPY_CONTAINER_VERSION=$VER" > $GITHUB_ENV
```

Since you're already in the "calculation job", use the same pattern and set the outputs. You could copy-paste what's in the artifact-name job:
```suggestion
    - name: Set PROXYPY_CONTAINER_VERSION env
      shell: bash
      run: >-
        VER=$(echo '${{
            steps.request-check.outputs.release-requested == 'true'
            && github.event.inputs.release-version
            || steps.scm-version.outputs.dist-version
        }}' | tr + .);
        echo "::set-output name=container-version::$VER"
```

Then, add a mapping to the `outputs:` section and you'll be able to use it in other jobs via the `needs` context.

---

Gotha, makes sense, thank you :)